modules/client/src/client-common/constants.ts

```diff
@@ -21,34 +21,54 @@ export const UNAVAILABLE_PROPOSAL_METADATA: ProposalMetadata = {
   resources: [],
 };
 
-const getGraphqlNode = (netowrk: string): string => {
+const getGraphqlNode = (netowrk: SupportedNetworks): string => {
   return `https://subgraph.satsuma-prod.com/${
     process.env.SATSUMA_API_KEY || "qHR2wGfc5RLi6"
-  }/aragon/${netowrk}/api`;
+  }/aragon/osx-${SupportedNetworksToGraphqlNetworks[netowrk]}/api`;
+};
+
+type GraphqlNetworks = "mainnet" | "goerli" | "polygon" | "mumbai";
+
+const SupportedNetworksToGraphqlNetworks: {
+  [K in SupportedNetworks]: GraphqlNetworks;
+} = {
+  homestead: "mainnet",
+  goerli: "goerli",
+  matic: "polygon",
+  maticmum: "mumbai",
```

Comment:
Wasn't the value osx-mainnet, osx-goerli... before the last change?

Reply:
Yes but all were preceeded by `osx-`, so I moved to the url constant, I can change itr back if you prefer it